### ISSUE TITLE
Are you able to install github copilot chat in vscode - Dev Spaces 15?

### ISSUE THREAD

**Issue by APINYK** [eclipse/che#23115]
### Summary

Hi, 

I am trying to install the github copilot extension as well as the github copilot chat extension in Dev Spaces 3.15. The copilot extension seems to be working as expected, however the chat extension does not. I have been testing in the redhat sandbox environment and get the error: 

![image](https://github.com/user-attachments/assets/6d3db22b-e350-45e0-86d4-4cdc47202a9b)

However that does not make much sense to me as it as been working on github codespaces for a long time and testing today codespaces is version 1.92.2 whereas the version on Dev Spaces 15 is 1.92.0.

I downloaded the extensions from vscode marketplace and did a manual install with "Install from VSIX" from the command palette.
[The github copilot chat extension-version that i have tried to install is version: 0.20.2024082301](https://marketplace.visualstudio.com/_apis/public/gallery/publishers/GitHub/vsextensions/copilot-chat/0.20.2024082301/vspackage).

Reading from earlier issues it seems that it has been possible to install the extension in the environment before: 
https://github.com/eclipse-che/che/issues/22781

I hope you can help me clarify this issue, thanks!

**Comment by akurinnoy** [comment]
Hi @vitaliy-guliy , could you please have a look?

**Comment by vitaliy-guliy** [comment]
I believe it is possible.
Only one issue I see here - is to download the latest working extensions binary for the current editor (for Dev Spaces 3.15 it is 1.92.0)
Before, I fetched a json with all possible extension versions and then downloaded the binaries by hands (beginning from the latest published version) and checked whether the extension works. If the extension didn't work, I downloaded previous version. And repeated the process until the extensions become working. Among this I checked the target field in the extension manifest. It determines the engine version (editor version).

Now I'm trying to write a script to fetch the latest working binary automatically. But it needs some time on investigating.
